test/unit/math/rev/scal/prob/normal_cdf_log_test.cpp

```diff
@@ -243,33 +243,44 @@ void test_value_and_derivatives(double expected_val, double y_dbl,
   EXPECT_FALSE(is_nan(gradients[0]));
   EXPECT_FALSE(is_nan(gradients[1]));
   EXPECT_FALSE(is_nan(gradients[2]));
+  // use relative error check in first case as logs of functions do not
+  // necessarily converge to asymptotic values
   if (!is_nan(gradients[0])) {
-    if (!is_nan(finite_diffs[0]))
-      EXPECT_NEAR(finite_diffs[0], gradients[0], 1e-2);
-    else
+    if (!is_nan(finite_diffs[0])) {
+      if (abs((finite_diffs[0] - gradients[0]) / finite_diffs[0]) > 0.1) {
```

Comment:
I guess these are the same sorts of tests as above. Just dropping a comment here so I remember them.

Reply:
I've changed this to a 1% relative error check as well